### ISSUE TITLE
Sprockets error

### ISSUE THREAD

**Issue by edsu** [sul-dlss/sul-embed#1429]
### Expected behavior

We should be able to follow the instructions in the README and run sul-embed in a development environment 😸

### Actual behavior

Requests for sul-embed are returning a Sprockets error:

```
Sprockets::Rails::Helper::AssetNotPrecompiledError in Embed#iframe

Showing /Users/edsummers/Projects/sul-dlss/sul-embed/app/views/embed/body/_media.html.erb where line #2 raised:

Asset `media.css` was not declared to be precompiled in production.
Declare links to your assets in `app/assets/config/manifest.js`.

  //= link media.css

and restart your server
```

<img width="1699" alt="Screenshot 2023-01-20 at 6 47 27 AM" src="https://user-images.githubusercontent.com/33829/213688458-6723ac85-e7dd-40b9-8b6d-341f9dd3cf43.png">

### Steps to reproduce the behavior

1. bin/rails server
2. visit http://localhost:3000/pages/sandbox/
3. submit the form
4. look at the error

### Browser / Environment

Firefox / MacOS

**Comment by edsu** [comment]
It looks like running `bin/rails assets:precompile` populates `app/assets/builds` which was empty for me before?

**Comment by edsu** [comment]
Fixed in 29f85d75bef76aadc2eec7e213dbd1a4c79ccc95
